js/client-battle.js

```diff
@@ -11,7 +11,7 @@
 
 			this.isSideRoom = Tools.prefs('rightpanelbattles');
 
-			this.$el.addClass('ps-room-opaque').html('<div class="battle">Battle is here</div><div class="foehint"></div><div class="battle-log"></div><div class="battle-log-add">Connecting...</div><div class="battle-controls"></div><button class="battle-chat-toggle button" name="showChat"><i class="fa fa-caret-left"></i> Chat</button>');
+			this.$el.addClass('ps-room-opaque').attr('role', 'main').html('<div class="battle">Battle is here</div><div class="foehint"></div><div class="battle-log" role="complementary" aria-label="Battle Log"></div><div class="battle-log-add">Connecting...</div><div class="battle-controls" role="complementary" aria-label="Battle Controls"></div><button class="battle-chat-toggle button" name="showChat"><i class="fa fa-caret-left"></i> Chat</button>');
```

Comment:
Think the best solution here is to edit the container so that its label is constructed from the participants of the battle or chat. For e.g, `aria-label="Battle with foo"`, `aria-label="Chat with foo", and add an appropriate role.

What'd be the easiest way to do this?